azure-pipelines.yml

```diff
@@ -5,20 +5,24 @@ jobs:
 - template: .azure/default-build.yml
   parameters:
     agentOs: Windows
+    cMakeRunArgs: '-A x64'
     beforeBuild:
-    - script: echo Hello, world!
-      displayName: 'Run a one-line script'
+    - powershell: "& vcpkg.exe install cpprestsdk cpprestsdk:x64-windows --vcpkg-root ./submodules/vcpkg"
+      displayName: vcpkg install dependencies
```

Comment:
Can we include this beforeBuild steps, extra cMakeRunArgs and the like for each OS in the README.md?

Reply:
I would also be good to include instructions to run `git submodule update --init` and whatever build scripts we decide to add.

Reply:
Oh duh, I meant to add a readme...

Reply:
y'know what's even better than a README (though there should be one as well)? `build.cmd` ;)